store/localstore/engine/engine.go

```diff
@@ -39,6 +39,9 @@ type DB interface {
 	// Seek searches for the first key in the engine which is >= key in byte order, returns (nil, nil, ErrNotFound)
 	// if such key is not found.
 	Seek(key []byte) ([]byte, []byte, error)
+	// SeekReverse searches for the first key in the engine which is less than key in byte order, returns (nil, nil, ErrNotFound)
```

Comment:
"SeekReverse searches the engine in backward order for the first key-value pair which key is less than the key in byte order, returns (nil, nil, ErrNotFound)" is better?